src/parsers/manifestjson.js

```diff
@@ -387,14 +404,25 @@ export default class ManifestJSONParser extends JSONParser {
   _validate() {
     // Not all messages returned by the schema are fatal to Firefox, messages
     // that are just warnings should be added to this array.
-    const warnings = [
+    let warnings = [
       messages.MANIFEST_PERMISSIONS.code,
       messages.MANIFEST_OPTIONAL_PERMISSIONS.code,
       messages.MANIFEST_HOST_PERMISSIONS.code,
       messages.MANIFEST_PERMISSION_UNSUPPORTED,
       messages.MANIFEST_FIELD_UNSUPPORTED,
     ];
 
+    // If the extension is already signed, just warn about privileged permissions
+    // and privileged properties (add-ons signed as privileged are expected to be
+    // submitted to AMO as already signed if they need to be listed).
```

Comment:
Do we have something like this already? I am wondering if this isn't something AMO already takes into account (e.g., maybe some errors are skipped).

Reply:
@diox